x/dualstaking/keeper/hooks.go

```diff
@@ -152,7 +152,10 @@ func (h Hooks) BeforeDelegationRemoved(ctx sdk.Context, delAddr sdk.AccAddress,
 	}
 	validator, err := h.k.stakingKeeper.GetDelegatorValidator(ctx, delAddr, valAddr)
 	if err != nil {
-		return nil
+		return utils.LavaFormatWarning("delegation removed hook failed", err,
+			utils.Attribute{Key: "validator_address", Value: valAddr.String()},
+			utils.Attribute{Key: "delegator_address", Value: delAddr.String()},
```

Comment:
You can use `utils.LogAttr` instead, it's shorter

Reply:
done. see https://github.com/lavanet/lava/commit/133f0e1fb5a46436cdcb891805b998fe1027074f